File: src/main/java/bot/utils/EventUtils.java
package bot.utils;

import net.dv8tion.jda.api.events.message.MessageReceivedEvent;

public final class EventUtils {
    public static String rawContent(MessageReceivedEvent event) {
        return event.getMessage().getContentRaw();
    }

    public static String[] spitRawContent(MessageReceivedEvent event) {
        return rawContent(event).split("\\s");
    }

    public static void sendMessageToChannel(MessageReceivedEvent event, String msg) {
        event.getChannel().sendMessage(msg).queue();
    }
}

File: src/main/java/bot/utils/EventUtils.kt
package bot.utils

import net.dv8tion.jda.api.events.message.MessageReceivedEvent

object EventUtils {
    fun rawContent(event: MessageReceivedEvent): String {
        return event.message.contentRaw
    }

    fun spitRawContent(event: MessageReceivedEvent): List<String> {
        return rawContent(event).split("\\s".toRegex())
    }

    fun sendMessageToChannel(event: MessageReceivedEvent, msg: String?) {
        event.channel.sendMessage(msg!!).queue()
    }
}